libs/platform/table/table.component.html

```diff
@@ -18,7 +18,7 @@
         }
     </div>
     @if (showGrowingButton) {
-        <fdp-table-growing-button [showItemsCount]="showItemsCount" [showItemsCount]="true"></fdp-table-growing-button>
```

Comment:
Removed a duplicate `showItemsCount` input. One instance was hardcoded to `true`, while the other took its value from the component input. Now, `showItemsCount` consistently derives its value from the component input.
@mikerodonnell89 @InnaAtanasova